papyrus-details.html

```diff
@@ -15,28 +15,35 @@
     <div id="summary-container">
       <content select="summary"/>
     </div>
-    <a id="toggle" on-tap="toggle" href="javascript:void(0)">[[_getLinkText(expanded,language)]]</a>
+    <a id="toggle" on-tap="toggle" href="javascript:void(0)">[[_getLinkText(expanded,lang)]]</a>
     <div id="expander" class$="[[_getHiddenClass(expanded)]]">
         <content/>
     </div>
   </template>
 
   <script>
-    (function(){
-        var languages = {
-            en: { open: "Open", close: "Close" },
-            fr: { open: "Ouvrir", close: "Fermer" },
-            de: { open: "Öffnen", close: "Schließen" },
-            es: { open: "Abrir", close: "Cerrar" }
-        };
+      PapyrusDetails = {};
+      PapyrusDetails.resources = {
+          en: { open: "Open", close: "Close" },
+          fr: { open: "Ouvrir", close: "Fermer" },
+          de: { open: "Öffnen", close: "Schließen" },
+          es: { open: "Abrir", close: "Cerrar" }
+      };
 
+        /**
+@demo demo/index.html
+        */
         Polymer({
           is: 'papyrus-details',
-          behaviors: [ LanguageBehavior ],
+          behaviors: [
+              PropertiesFromAncestorBehavior({
+                    lang: { defaultValue: 'en' },
```

Comment:
I see. HTML elements do [have a `lang` property](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/lang).

Well, calling it 'lang' here is exactly deliberate because we explicitly want it to get its value from [the very standard attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang), so we have a little conflict of intentions.

I understand the concern but I haven't found an issue with declaring it in polymer so far, so let's think of one.  
I actually have a feeling this might be benign, at least in most cases, because the only problematic situation I can think of, is for when we want to listen to the changes from such native properties (Polymer won't be able to receive direct sets, hence it needs the clue of `...::eventName`). Here, the native property is overridden only to set its value, as we don't support being able to change the language from the descendant.